### PR TITLE
Update README.mkd for responsive snippet

### PR DESCRIPTION
**Pull request by thecodemensch**

### CODE REVIEW

**Comment by mattn** [comment]
`html:5` does not expand meta viewport on https://docs.emmet.io/

**Comment by proprytr** [comment]
> `html:5` does not expand meta viewport on https://docs.emmet.io/
Well Sir, thanks for this awesome vim-plugin, btw what we can do to have this functionality, any way out?

**Comment by mattn** [comment]
Please try to define your snippets.

```vim
let g:user_emmet_settings = {
\  'variables': {'lang': 'ja'},
\  'html': {
\    'default_attributes': {
\      'option': {'value': v:null},
\      'textarea': {'id': v:null, 'name': v:null, 'cols': 10, 'rows': 10},
\    },
\    'snippets': {
\      'html:5': "<!DOCTYPE html>\n"
\              ."<html lang=\"${lang}\">\n"
\              ."<head>\n"
\              ."\t<meta charset=\"${charset}\">\n"
\              ."\t<title></title>\n"
\              ."\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n"
\              ."</head>\n"
\              ."<body>\n\t${child}|\n</body>\n"
\              ."</html>",
\    },
\  },
\}
```

**Comment by proprytr** [comment]
> Please try to define your snippets.
> 
> ```viml
> let g:user_emmet_settings = {
> \  'variables': {'lang': 'ja'},
> \  'html': {
> \    'default_attributes': {
> \      'option': {'value': v:null},
> \      'textarea': {'id': v:null, 'name': v:null, 'cols': 10, 'rows': 10},
> \    },
> \    'snippets': {
> \      'html:5': "<!DOCTYPE html>\n"
> \              ."<html lang=\"${lang}\">\n"
> \              ."<head>\n"
> \              ."\t<meta charset=\"${charset}\">\n"
> \              ."\t<title></title>\n"
> \              ."\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n"
> \              ."</head>\n"
> \              ."<body>\n\t${child}|\n</body>\n"
> \              ."</html>",
> \    },
> \  },
> \}
> ```

It worked, You should update this on README.mkd.

File: plugin/emmet.vim [hunk 1]
```diff
@@ -23,6 +23,7 @@
 "      | <head>
 "      |     <title></title>
 "      |     <meta charset="UTF-8">
+"      |     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (mattn) [inline plugin/emmet.vim:26]: Please revert this.

File: README.mkd [hunk 2]
```diff
@@ -110,6 +110,32 @@ You can change the **path** to your **snippets_custom.json** according to your p
...
+	\              ."\t<title></title>\n"
+	\              ."\t<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n"
+	\              ."</head>\n"
+	\              ."<body>\n\t${child}|\n</body>\n"
+	\              ."</html>",
+	\    },
+	\  },
+	\}
+
+
```
Comment (mattn) [inline README.mkd:138]: This is okay. Thank you.

**Comment by mattn** [comment]
Thank you

**Comment by thecodemensch** [comment]
No need of it, always up for open source :)

**Pull request merged**